File: 771.jewels-and-stones.js
/*
 * @lc app=leetcode id=771 lang=javascript
 *
 * [771] Jewels and Stones
 */

// @lc code=start
var numJewelsInStones = function(jewels, stones) {
  jewels = [...jewels]
  return [...stones].filter(v=>jewels.includes(v)).length
};
// @lc code=end

File: 771.jewels-and-stones.py
#
# @lc app=leetcode id=771 lang=python3
#
# [771] Jewels and Stones
#

# @lc code=start
class Solution:
    def numJewelsInStones(self, jewels: str, stones: str) -> int:
        return len([stone for stone in stones if stone in jewels])
# @lc code=end

